tensorrt_llm/runtime/kv_cache_manager_v2/_core/_kv_cache_manager.py

```
@@ -687,15 +796,17 @@ def get_num_slots(seq_len: int) -> TypedIndexList[PoolGroupIndex, int]:
 
         for pg in typed_range(num_pool_groups):
             remaining_slots[pg] -= get_num_slots(1)[pg] * (batch_size - 1)
-            assert remaining_slots[pg] >= 0
+            if remaining_slots[pg] < 0:
+                return 0
 
         def is_enough(num_blocks: int) -> bool:
             return all(
                 cnt <= rem
                 for cnt, rem in zip(get_num_slots(num_blocks * tokens_per_block), remaining_slots)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)